### DISCUSSION TITLE
New source for ffmpeg in the wiki install instructions is needed (Linux)

### DISCUSSION THREAD

**Discussion by Goldenbeasty** [alvr-org/ALVR#1587] [General]
The ALVR [Linux (other) install instructions](https://github.com/alvr-org/ALVR/wiki/Installation-guide#other) refers to [ppa:savoury1/ffmpeg5](https://launchpad.net/~savoury1/+archive/ubuntu/ffmpeg5), a PPA, that has recently converted over to a "subscriber only" model, requiring a payment/donation to the maintainer for successful installation.

```
*** Several PPAs now need the new private PPA for successful installation:
Blender; HandBrake; FFmpeg 5, 6 & git; mpv & mpv-git; and VLC 3. Any other PPAs, with many very useful upgrades, are still freely usable by all. ***
```
See https://launchpad.net/~savoury1

```
FFmpeg 5.1.3 builds (also requires ppa:savoury1/ffmpeg) for Xenial & newer.

*** FFmpeg builds here require the private PPA to successfully install! ***
```
See https://launchpad.net/~savoury1/+archive/ubuntu/ffmpeg5

A list of ffmpeg build flags (configuration), while not being the best solution, would not depend on any direct source and would help to validate that the ffmpeg used is compatible with ALVR.

**Comment by zmerp** [comment]
ALVR doesn't not need any system-wide ffmpeg installation anymore. I will update the wiki.
